Kaptain 2.0.0 Docs sdk folder (#4319)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/sdk/0.3.x/quickstart/index.md

---
layout: layout.pug
navigationTitle: Quick Start
title: Quick Start
beta: false
menuWeight: 0
---

The Kaptain SDK provides a high-level API to support model development workflows and deals with Kubernetes specifics for the
users' benefit. The main building blocks required for model development are a user-provided model definition,
training source code, and a configured instance of the `Model` class from the SDK.

The SDK automatically packs user-provided files into a Docker image which is then used in the model training and
tuning. The SDK also provides utilities to simplify the export of the trained models to the supported storage locations.

# Using the SDK from a Jupyter Notebook

## Configure Docker credentials

In order to use the SDK from a Jupyter Notebook, you must configure credentials for accessing a Docker
registry. It is recommended to distribute registry configuration and credentials by following the
[Accessing Docker and Cloud Storage][credentials] guide.

> However, to hit the ground running, it is also sufficient to create a valid Docker credentials file from a notebook

Reviewer: ```suggestion
However, for a quick start, it is also sufficient to create a valid Docker credentials file from a notebook
```
